### ISSUE TITLE
[Sonic Blox] Game view unexpectedly moves

### ISSUE THREAD

**Issue by Toad06** [ruffle-rs/ruffle#792]
https://www.newgrounds.com/portal/view/190480/
In this Tetris clone, the game view moves after a block has hit the ground (or another block). This then remains permanent.

![s1](https://user-images.githubusercontent.com/14274782/86351240-3b8bf180-bc64-11ea-9f48-b44e2e651a7f.png) => ![s2](https://user-images.githubusercontent.com/14274782/86351253-3f1f7880-bc64-11ea-8b37-cc1f47599042.png) => ![s3](https://user-images.githubusercontent.com/14274782/86351276-4a72a400-bc64-11ea-9b6c-79a5d4f3b8ac.png)

On Flash, the block just changes its color to gray.
![se](https://user-images.githubusercontent.com/14274782/86351541-b6eda300-bc64-11ea-84b3-0ee46c7f40b6.png)

The issue seems to be in frame 21:
```
_root.attachMovie("singlebrick","singlebrick" + j + i,j * 22 + i);
with(eval("_root.singlebrick" + j + i))
{
    _y = (j - 2) * 25
    _x = (i + 2) * 25
};
```

**Comment by Herschel** [comment]
The bugs you find are always really funny!

**Comment by Herschel** [comment]
Fixed by #803.
